bin/pygrb/pycbc_pygrb_plot_chisq_veto

```diff
@@ -20,197 +20,332 @@
+    colors = ["k-" if snr == new_snr_thresh else
+              "y-" if snr == int(snr) else
+              "y--" for snr in new_snrs]
+
+    return colors
 
+
+# =============================================================================
+# Main script starts here
+# =============================================================================
 description = 'Produces signal consistency plots of the form SNR versus veto.'
 usage = __program__ + ' [--options]'
-opts = pygrb_plot_opts_parser(usage=usage, description=description, version=__version__)
 
-if opts.verbose:
-    level = logging.INFO
-else:
-    level = logging.WARNING
-logging.basicConfig(format="%(asctime)s:%(levelname)s : %(message)s",
-                    level=level)
+parser = ppu.pygrb_initialize_plot_parser(usage=usage, description=description,
+                                          version=__version__)
+parser.add_argument("-t", "--trig-file", action="store",
+                    default=None, required=True,
+                    help="The location of the trigger file")
+parser.add_argument("-z", "--zoom-in", default=False, action="store_true",
+                    help="Output file a zoomed in version of the plot.")
+parser.add_argument("-y", "--y-variable", required=True, help="Quantity " +
+                    "to plot on the vertical axis. Supported choices " +
+                    "are: standard, bank, or auto.")
+parser.add_argument("--use-sngl-ifo-snr", default=False,
+                    action="store_true", help="Plots are vs single IFO " +
+                    "SNR, rather than coherent SNR")
+parser.add_argument("--use-sngl-ifo-veto", default=False,
+                    action="store_true", help="Single IFO veto values " +
+                    "plotted, rather than network ones")
+parser = ppu.pygrb_add_bestnr_opts(parser)
+opts = parser.parse_args()
 
+init_logging(opts.verbose, format="%(asctime)s: %(levelname)s: %(message)s")
+
+# Check options
 trig_file = os.path.abspath(opts.trig_file)
 inj_file = None
 if opts.inj_file:
     inj_file = os.path.abspath(opts.inj_file)
-outfile = opts.output_file
-zoomedoutfile = opts.zoomed_output_file
-if zoomedoutfile is None:
-    msg = "Please specify a pathname with the zoomed-ouptut-file option"
-    logging.error(msg)
-    sys.exit()
+zoom_in = opts.zoom_in
 veto_files = []
 if opts.veto_directory:
-    veto_string = ','.join([str(i) for i in range(2,opts.veto_category+1)])
+    veto_string = ','.join([str(i) for i in range(2, opts.veto_category+1)])
     veto_files = glob.glob(opts.veto_directory +'/*CAT[%s]*.xml' %(veto_string))
-# If an ifo is given, the veto is intended as a single IFO quantity
+# Check that the y-axis variable was specified correctly
+if opts.y_variable in ['standard', 'bank', 'auto']:
+    veto_type = opts.y_variable
+else:
+    err_msg = 'Unrecognized option given with --y-variable'
+    logging.error(err_msg)
+    raise ValueError(err_msg)
+veto_type = opts.y_variable
```

Comment:
This block is not necessary, just use the `choices` kwarg in `parser.add_argument()`.